libtextsecure/contacts_parser.js

```diff
@@ -14,7 +14,7 @@ ProtoParser.prototype = {
       if (this.buffer.limit === this.buffer.offset) {
         return undefined; // eof
       }
-      const len = this.buffer.readVarint32();
+      const len = this.buffer.readInt32();
```

Comment:
did this break on mobile?

Reply:
Yes this broke iOS I believe, this seems to work on all platforms now tho

Reply:
Nice catch

Reply:
Oh no it was Niels/Mik, I was just following orders :nerd_face: